lib/qs/payload.rb

```diff
@@ -0,0 +1,47 @@
+require 'qs'
+require 'qs/job'
+
+module Qs
+
+  module Payload
+
+    def self.deserialize(serialized_payload)
+      self.deserialize_hash(Qs.deserialize(serialized_payload))
+    end
+
+    def self.deserialize_hash(payload_hash)
+      Qs::Job.new(payload_hash['name'], payload_hash['params'], {
+        :type       => payload_hash['type'],
+        :created_at => Time.at(payload_hash['created_at'].to_i)
+      })
+    end
+
+    def self.serialize(job)
+      self.serialize_hash({
+        'type'       => job.payload_type.to_s,
+        'name'       => job.name.to_s,
+        'params'     => StringifyParams.new(job.params),
+        'created_at' => job.created_at.to_i
+      })
+    end
```

Comment:
@jcredding the serialize isn't very symmetrical with the deserialize method, but I feel like they *should* be (one should just be doing the opposite of the other.